torch/legacy/nn/MultiLabelSoftMarginCriterion.py

```diff
@@ -18,8 +18,10 @@ class MultiLabelSoftMarginCriterion(Criterion):
 
     """
 
-    def __init__(self, weights=None):
+    def __init__(self, weights=None, sizeAverage=True):
         super(MultiLabelSoftMarginCriterion, self).__init__()
+        self.weights = weights
+        self.sizeAverage = sizeAverage
```

Comment:
Can you add a comment saying that this does nothing?
If I'm understanding this correctly, if sizeAverage does nothing, why are you assigning it? Does assigning weights to `self.weights` do anything either? Could you have assigned `None` to it?

Reply:
I actually did this to make test/common_nn.py pass for legacy when I add the reference function. It doesn't do anything other than that. Do you think that's okay or should I find another way to work around it?

Reply:
Do the tests actually use the weights / sizeAverage attributes for the legacy tests?